libr/magic/d/default/filesystems

```diff
@@ -110,6 +110,16 @@
 >>96	string	read\ error\ while\ reading\ drive	\b, FREE-DOS Beta 0.9 MBR
 >271	string	Operating\ system\ loading
 >>296	string	error\r					\b, SYSLINUX MBR (2.10)
+<<<<<<< HEAD
+# https://www.acronis.de/
+>362	string	MBR\ Error\ \0\r			
+>>376	string	ress\ any\ key\ to\ 			
+>>>392	string	boot\ from\ floppy...\0			\b, Acronis MBR
+# added by Joerg Jenderek
+# https://www.visopsys.org/
+# https://partitionlogic.org.uk/
+>309	string	No\ bootable\ partition\ found\r	
+=======
```

Comment:
conflicts, this is a broken rebase/merge pr, please fix that

Reply:
Hey how to do that? I am actually new